### ISSUE TITLE
Also set group ID

### ISSUE THREAD

**Issue by sholladay** [sindresorhus/downgrade-root#4]
Shouldn't this module also be doing `process.setgid()` in addition to `process.setuid()`?

This code ...

``` js
// foo.js
'use strict';

require('downgrade-root')();
console.log('uid:', process.getuid());
console.log('gid:', process.getgid());
```

... prints `501` and `0` for me when run with `sudo node foo` on OS X.

The former is correct, the later is not what I expected.

Is this an oversight or an intentional decision? The tests don't give any guidance on that. Would be nice to have tests for `gid` either way.

**Comment by sindresorhus** [comment]
I think it's an oversight.

@silverwind Thoughts?

**Comment by silverwind** [comment]
Indeed, I added support for downgrading to `SUDO_GID`. We could do a fallback to default group ids after some research for default GIDs. FWIW, it seems to be 20 on OSX, 100 on Arch Linux, and something above 1000 on Debian.

**Comment by sholladay** [comment]
Thank you! I appreciate it.

**Comment by silverwind** [comment]
Published as 1.2.0.
